lib/cucumber/factory.rb

```diff
@@ -88,6 +89,12 @@ def parse_creation(world, raw_model, raw_variant, raw_attributes, raw_boolean_at
             attributes[attribute] = flag
           end
         end
+
+        if multiline_attributes.present?
+          multiline_parser = MultilineParser.new(multiline_attributes)
+          attributes = attributes.merge(multiline_parser.extract_attributes)
+        end
+
```

Comment:
I wonder if it is worth it to slightly rework this, so that the `#attribute_value` method is also applied on the multiline arguments. That way it would be possible to set associations and use cucumber tranforms for the multiline syntax.